packages/ketcher-core/src/infrastructure/services/struct/RemoteStructServiceProvider.ts

```diff
@@ -14,18 +14,30 @@
  * limitations under the License.
  ***************************************************************************/
 
+import { StructService, StructServiceOptions } from './structService.types'
+
 import {
-  StructService,
   StructServiceProvider,
   ServiceMode
-} from './structService.types'
-import IndigoService from './remoteStructService'
+} from './structServiceProvider.types'
+import RemoteStructService from './RemoteStructService'
 
 class RemoteStructServiceProvider implements StructServiceProvider {
+  private apiPath: string
   mode: ServiceMode = 'remote'
 
-  createStructService(baseUrl: string, options: any): StructService {
-    return new IndigoService(baseUrl, options)
+  constructor(baseUrl: string) {
```

Comment:
It would be better to call as apiPath everywhere

Reply:
fixed